src/docs/sdk/performance/dynamic-sampling-context.mdx

```diff
@@ -0,0 +1,158 @@
+
+For the propagation of DSC, SDKs first read the DSC from the baggage header of incoming requests/messages.
+To propagate DSC to downstream SDKs/services, we create a baggage header (or modify an existing one) through HTTP request instrumentation.
+
+<Alert level="info">
+
+Other vendors might also be using the `baggage` header.
+If a `baggage` header already exists on an outgoing request, SDKs should aim to be good citizens by only **appending** Sentry values to the header.
+In the case that another vendor added Sentry values to an outgoing request, SDKs may overwrite those values.
+
+SDKs must not add other vendors' baggage from incoming requests to outgoing requests.
+Sentry SDKs only concern themselves with Sentry baggage.
+
+</Alert>
+
+The following is an example of what a baggage header containing Dynamic Sampling Context may look like:
+
+```
+baggage: other-vendor-value-1=foo;bar;baz, sentry-traceid=771a43a4192642f0b136d5159a501700, sentry-publickey=49d0f7386ad645858ae85020e393bef3; sentry-userid=Am%C3%A9lie, other-vendor-value-2=foo;bar;
+```
+
+See the [Payloads section](#payloads) for a complete list of key-value pairs that SDKs should propagate.
+
+## Payloads
+
+Dynamic Sampling Context is propagated via a baggage header and sent to Sentry via <Link to="/sdk/envelopes/#transaction">transaction envelope headers</Link>.
+
+### Baggage-Header
+
+SDKs may set the following key-value pairs on baggage headers.
+While all of these values are optional, SDKs should make their best effort to add as many of them to the baggage header as possible when starting a trace.
+
+- `sentry-traceid` - The original trace ID as generated by the SDK
+- `sentry-publickey` - Public key as defined by the user via the DSN in the SDK options
+- `sentry-release` - The release as defined by the user in the SDK options
+- `sentry-environment` - The environment as defined by the user in the SDK options
+- `sentry-transaction` - The name of the trace's origin transaction in unparameterized (raw) format
+- `sentry-userid` - User ID as set by the user with <Link to="/sdk/unified-api/#scope">`scope.set_user`</Link>
+- `sentry-usersegment` - User segment as set by the user with <Link to="/sdk/unified-api/#scope">`scope.set_user`</Link>
+- `sentry-samplerate` - Sample rate as defined by the user in the SDK options
+
+SDKs must set all of the keys in the form of "`sentry-[name]`".
+The delimiter "`sentry-`" acts to identify key-value pairs set by Sentry SDKs.
+Additionally, we chose `[name]` to be written in "snake case" without any underscore ( `_` ) characters. This naming convention is the most language agnostic.
+
+### Envelope Header
+
+Dynamic Sampling Context is transferred to Sentry through the <Link to="/sdk/envelopes/#transaction">transaction envelope headers</Link>, keyed by `trace`.
+It corresponds directly to the definition of <Link to="/sdk/performance/trace-context/#trace-context">Trace Context</Link>.
```

Comment:
What's the urgency on renaming TraceContext to Dynamic Sampling Context for SDKs?

Reply:
I think those are still more or less different concepts. The word `Dynamic Sampling Context` represents the sentry-* values we propagate in the baggage header. The term `TraceContext` is incredibly overloaded - in the docs I just wrote it simply refers to the object schema over at https://develop.sentry.dev/sdk/performance/trace-context/#trace-context

I know in SDKs there are some fields called for example`event.contexts.trace` but those fields are something else again. Right now we only care about Dynamic Sampling Context propagation in baggage and `TraceContext` in the event envelope header.

Reply:
I was referring to this comment https://github.com/getsentry/develop/issues/611#issuecomment-1161929944 and comments following it.

Reply:
Sorry to jump in here but what we proposed was to find a good term for the data we want to *both*, propagate to downstream SDKs (i.e. via the baggage Http header) *and* send to Relay (via the `trace` envelope header). That data should be the same, however, it is structured differently (as pointed out in the docs added by this PR). 

The reasons for introducing "Dynamic Sampling Context" (DSC) were already mentioned: Trace Context is an overloaded term, we observed a lot of confusion around terms like "baggage", "trace state", "trace context", etc. and DSC should aim to unify this as much as possible. We're aware that Relay is currently calling the `trace` envelope header `TraceContext`. IMO it's not as important to change this right now but to rather have a term we can all agree on when talking about _what_ we actually propagate/send.  

Discussed this with @lforst and we agree on that

Reply:
So DSC is what we use to internally to transport tracing information from an incoming request that may exist or the place it where the tracing information is created in the SDK (that's first in line) to the outgoing request (be that an API call or an envelope sent to Sentry).

Would it make sense to find a name for the "first in line" SDK to more easily refer to it? e.g. head SDK?